{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -13,14 +13,16 @@ BUCKET = {{ cookiecutter.s3_bucket }}
 PROFILE = {{ cookiecutter.aws_profile }}
 PROJECT_NAME = {{ cookiecutter.repo_name }}
 PYTHON_INTERPRETER = {{ cookiecutter.python_interpreter }}
+IMAGE_VERSION = {{ cookiecutter.base_image_version }}
 SLASH = /
```

Comment:
Is there any better way to do what I'm doing with this SLASH variable? It seems like duct tape to me.

Reply:
I seem to recall your trying `docker build --build-arg directory=/$(PROJECT_DIR)` previously -- if so and it didn't work, the answer is beyond my immediate reckoning .